setup.cfg

```diff
@@ -0,0 +1,18 @@
+[metadata]
+name = py-wkhtmltox
+version = 0.3
+author = Matt Reiferson
+author_email = mreiferson@gmail.com
+description = Python bindings for libwkhtmltox
+url = http://github.com/mreiferson/py-wkhtmltox
+
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, <3.5
```

Comment:
Do not restrict to `<3.5`, we don't want to update the library if it doesn't require changes for higher versions

Reply:
done